packages/mgt-chat/src/statefulClient/GraphNotificationUserClient.ts

```diff
@@ -304,13 +328,11 @@ export class GraphNotificationUserClient {
       withCredentials: false
     };
 
-    // retry with the following intervals, the last interval will take precedence if there are more retries than intervals
-    const retryTimes = [0, 2000, 10000, 30000, 45000, 60000, 90000, 120000, 180000, 240000];
+    // retry with the following intervals, else give up by returning null, we keep the retry interval pretty short and only 2 times
+    const retryTimes = [10, 100];
```

Comment:
One thing I don't know about re: this. Can multiple connections be made to the same signal-R endpoint from the same client. This is an automatic retry connection policy and so you run the risk of 2 connections if:
1. The client disconnects
2. 50ms later the renewal fires
3. A new client is created and tries to connect

You could force a test of this situation or drop the retry policy.

When we were talking about this originally I thought we were discussing retry only with start() but that doesn't appear to be the case.

Reply:
Why do we think it is possible to have concurrent renewals since we already gaurantee only one renewal ever happens at any one time?

Reply:
made renewal check more frequent instead